core/src/types.rs

```diff
@@ -200,12 +201,15 @@ pub enum ModelError {
 impl Model {
     pub fn new(input: &str) -> Result<Self, ModelError> {
         let mut parts: Vec<&str> = input.split('/').collect();
+        info!("{:?}", parts);
         let missing_source = parts.len() != 2;
         if missing_source && parts[0] == "text-embedding-ada-002" {
             // for backwards compatibility, prepend "openai" to text-embedding-ada-2
             parts.insert(0, "openai");
         } else if missing_source && parts[0] == "all-MiniLM-L12-v2" {
             parts.insert(0, "sentence-transformers");
+        } else if missing_source && parts[0] == "llama2" {
+            parts.insert(0, "ollama");
```

Comment:
this block of code is just for some backwards compatibility. previously, just providing "text-embedding-ada-002" would route the request to openai. We can remove it for Ollama, the user should provide `ollama/llama2`/

Reply:
Okk, removed this block in the recent commit